pkg/schema/type.go

```diff
@@ -13,13 +13,16 @@ import (
 
 const (
 	AnnotationSchemaNullable structmeta.AnnotationName = "schema/nullable"
+	AnnotationSchemaType     structmeta.AnnotationName = "schema/type"
+	SchemaTypeAny            string                    = "any"
```

Comment:
```suggestion
	TypeAnnotationKwargAny   string                    = "any"
```

Prior art:
https://github.com/vmware-tanzu/carvel-ytt/blob/b4a5e57e0b7b6d2d27edd857f8c8d3907172133b/pkg/yttlibrary/overlay/match_annotation_expects_kwarg.go#L16-L21